gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/handlers/accesspoint/manager/AccessPointManagerImpl.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.gateway.handlers.accesspoint.manager;
+
+import io.gravitee.gateway.handlers.accesspoint.model.AccessPoint;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public class AccessPointManagerImpl implements AccessPointManager {
+
+    private final Map<String, AccessPoint> accessPoints = new ConcurrentHashMap<>();
+
+    @Override
+    public void register(AccessPoint accessPoint) {
+        accessPoints.put(accessPoint.getReferenceId(), accessPoint);
```

Comment:
It could be a list of access point by environment.

Reply:
ah yes I forgot this because can only set one through cockpit but forgot about the templating which can make it more than one

Reply:
No, as the customer is able to override the value, and it also could have multiple gateway deployed which will lead to different access points.